### PR TITLE
Add to override_respot + panning_horizons wildcard

### PR DESCRIPTION
**Pull request by Eddy-JV**
# Closes # (if applicable).

## Changes proposed in this Pull Request

This PR is part of adding Params to all Rules in Snakemake under Issue https://github.com/pypsa-meets-earth/pypsa-earth-sec/issues/330

The following Rules are already done including the work done in this PR:

- [x] add_export.py
- [x] build_base_energy_totals.py
- [x] build_base_industry_totals.py
- [x] build_clustered_population_layouts.py -> _(Had no changes to be done)_
- [x] build_cop_profiles.py
- [x] build_heat_demand.py
- [x] build_industrial_database.py -> _(Had no changes to be done)_
- [x] build_industrial_distribution_key.py
- [x] build_industry_demand.py
- [x] build_population_layouts.py
- [x] build_ship_profile.py
- [x] build_solar_thermal_profiles.py
- [x] build_temperature_profiles.py
- [x] copy_config.py
- [x] helpers.py -> _(Had no changes to be done)_
- [x] make_summary.py
- [x] override_respot.py
plot_network.py
plot_network_eur.py
plot_summary.py
prepare_airports.py
prepare_db.py
prepare_energy_totals.py
prepare_gas_network.py
prepare_heat_data.py
prepare_ports.py
prepare_sector_network.py
prepare_transport_data.py
prepare_transport_data_input.py
prepare_urban_percent.py
solve_network.py

## Checklist

- [x] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
- [x] Newly introduced dependencies are added to `envs/environment.yaml` and `envs/environment.docs.yaml`.
- [x] Changes in configuration options are added in all of `config.default.yaml`, `config.tutorial.yaml`, and `test/config.test1.yaml`.
- [x] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: scripts/override_respot.py [hunk 1]
```diff
@@ -102,24 +102,7 @@ def override_values(tech, year, dr):
         else:
             print("No RES potential techs to override...")
 
-        if (
```
Comment (energyLS) [inline scripts/override_respot.py:105]: @Eddy-JV why is this deleted? Indeed, it does not look like nice code... But what did it do and do we still need it?

Comment (Eddy-JV) [reply scripts/override_respot.py:105]: I deleted it because it looks like it is only hard coded for Marocco. Which will lead to erros if another country is used if both  
 snakemake.config["custom_data"]["add_existing"]
 and snakemake.wildcards["planning_horizons"] == "2030"

are satisfied.

**Review by energyLS — APPROVED** [review]
